feedback/lib/src/l18n/translation.dart

```diff
@@ -235,10 +237,14 @@ class GlobalFeedbackLocalizationsDelegate
     const Locale('zh'): const ZhFeedbackLocalizations(),
   };
 
+  /// The default locale to use. Note that this locale should ALWAYS be
+  /// present in supportedLocales.
+  static const defaultLocale = Locale('de');
```

Comment:
Shouldn't this be `en`?
```suggestion
  static const defaultLocale = Locale('en');
```